_data/glossary/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,15 @@
+id: persistent-volume-claim
+name: Persistent Volume Claim
+full-link: /docs/concepts/storage/persistent-volumes/
+related:
+- persistent-volume
+- statefulset
+- deployment
+- pod
+tags:
+- core-object
+- storage
+short-description: >
+  A PersistentVolumeClaim (PVC) is used to claim storage resources defined in a PersistentVolume (PV) so that it can be mounted as a volume in a container.
```

Comment:
Link to the glossary item for `PV` once https://github.com/kubernetes/website/pull/6591 is merged:

`{% glossary_tooltip text="Persistent Volume" term_id="persistent-volume" %}`